projects/schematics/src/add-spartacus/index.ts

```diff
@@ -42,6 +53,12 @@ import {
 import { getProjectFromWorkspace } from '../shared/utils/workspace-utils';
 import { Schema as SpartacusOptions } from './schema';
 
+const DEFAULT_ENVIRONMENT_NAME = 'environment';
```

Comment:
Maybe we should keep it with rest of vars in constants.ts?

Reply:
Done